.github/workflows/pyright.yml

```diff
@@ -0,0 +1,44 @@
+name: Check types
+on: [pull_request]
+env:
+  NODE_VERSION: '18'
+  PYTHON_VERSION: '3.9'
+
+permissions:
+  contents: read
+
+jobs:
+  typecheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        id: gitcheckout
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Install python deps
+        id: pydepends
+        if: steps.gitcheckout.outcome == 'success'
+        run: |
+            python3 -m venv /home/runner/work/app
+            mkdir -p /home/runner/work/app/src/ckanext-openapiview
+            cp -R . /home/runner/work/app/src/ckanext-openapiview
+            source /home/runner/work/app/bin/activate
+            pip install --upgrade setuptools==70.0.0
+            pip install --upgrade pip==23.2.1
+            pip install -e 'git+https://github.com/open-data/ckan@canada-v2.10#egg=ckan' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/requirement-setuptools.txt' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/requirements.txt' -r 'https://raw.githubusercontent.com/open-data/ckan/canada-v2.10/dev-requirements.txt'
```

Comment:
this extension shouldn't need anything from our fork of ckan

Reply:
True, I don't think Power BI or GC Notify either require our fork too, so will update those.